src/widgets/ReportSideBar/ReportSideBar.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/no-unused-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, ListItem, FlatList, Text, StyleSheet } from 'react-native';
+import { APP_FONT_FAMILY, GREY } from '../../globalStyles';
+import { pageInfoStrings } from '../../localization/pageInfoStrings';
+
+export const ReportSidebar = props => {
+  const { dimensions, onPressItem, selectedItemIndex, data } = props;
```

Comment:
`export const ReportSidebar = ({ dimensions, onPressItem, selectedItemIndex, data }) => { ... }` 

just to be consistent 🤣

Reply:
Thanks!!

Reply:
Done here: a8b88cc
Thanks!